app/views/cancelCase/CancelCaseDisclosureReferenceNumberView.scala.html

```diff
@@ -41,20 +41,19 @@
                 classes = Some("govuk-!-width-one-half")
             )
 
-        @button("common.button.continue")
-    }
-
-    @govukDetails(
-        Details(
-            summary = HtmlContent(messages("cancelCase.disclosureReference.detailsHeader")),
-            content = HtmlContent(
-                messages("cancelCase.disclosureReference.details_p1") +
-                        """<a class="govuk-link" href="mailto:customsaccountingrepayments@hmrc.gov.uk">customsaccountingrepayments@hmrc.gov.uk</a>"""
-                        + messages("cancelCase.disclosureReference.details_p2")
+        @govukDetails(
+            Details(
+                summary = HtmlContent(messages("cancelCase.disclosureReference.detailsHeader")),
+                content = HtmlContent(
+                    messages("cancelCase.disclosureReference.details_p1") +
+                    """<a class="govuk-link" href="mailto:customsaccountingrepayments@hmrc.gov.uk">customsaccountingrepayments@hmrc.gov.uk</a>""" +
+                    messages("cancelCase.disclosureReference.details_p2")
+                )
             )
         )
-    )
 
+        @button("common.button.continue")
+    }
```

Comment:
this originally wasn't part of the story, but the cancel case reference number view needed to be updated too (this progressive disclosure was not placed correctly)